docs/source/models.rst

```diff
@@ -2331,11 +2331,53 @@ Example: Log a LangChain Agent
 .. literalinclude:: ../../examples/langchain/simple_agent.py
     :language: python
 
+
+Logging RetrievalQA Chains
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``langchain`` flavor in MLflow supports the logging of RetrievalQA chains, a feature that is not fully native to LangChain.
+This is achieved by supporting the serialization of the retriever object, a crucial step in creating a RetrievalQA chain.
+
+By default, LangChain serialization of a RetrievalQA chain does not include the retriever object. Instead, it mandates the
+provision of a retriever object during the loading phase. The MLflow langchain flavor, on the other hand, implements the
+serialization of this retriever object, thereby simplifying the process.
```

Comment:
Can we simplify the language / phrasing in the docs here? Would like to make sure it's as easy for users to understand as possible. I think 1-2 inline code examples would be a lot easier for users to follow than the ~ 10 sentence English description :)

Reply:
@dbczumar Updated, thanks